tonynatoci-annarbor.md

```diff
@@ -0,0 +1,7 @@
+## Ann Arbor, Mich.
+
+### Food
+
+### Places to take good Instagram pictures
+
+### Don't go here
```

Comment:
@francisvicta there aren't any dangerous spots in Ann Arbor, but it's a small town. If you venture beyond the borders into Ypsilanti or the surround towns, it can get a little shady. Avoid Washtenaw Avenue (US-23 Business) at night.